packages/es-components/src/components/containers/modal/Modal.js

```diff
@@ -41,17 +41,13 @@ const ModalDialogMedium = styled.div`
 `;
 
 const ModalDialogSmall = styled(ModalDialogMedium)`
-  width: 100%;
```

Comment:
All this looks ok when I run it, but I'm wondering what made it necessary to remove this width statement.

Reply:
Both Small and Large dialogs are based off Medium, which already has width: 100% set.